conda_forge_webservices/update_me.py

```diff
@@ -14,7 +15,7 @@
 
 LOGGER = logging.getLogger("conda_forge_webservices.update_me")
 
-PKGS = ["conda-smithy"]
+PKGS = ["conda-smithy", "conda", "conda-build"]
```

Comment:
Would it make sense to add

```suggestion
PKGS = ["conda-smithy", "conda", "conda-build", "conda-libmamba-solver", "mamba"]
```

Reply:
yes and thank you!